### PR TITLE
Fix an issue with non existing compileIncremental key

### PR DESCRIPTION
**Pull request by tgodzik**
Fixes https://github.com/scalacenter/bloop/issues/1158

### CODE REVIEW

File: integrations/sbt-bloop/src/main/scala-sbt-0.13/bloop/integrations/sbt/Offloader.scala [hunk 1]
```diff
@@ -15,7 +15,5 @@ object Offloader {
 
   val bloopCompileProjectSettings: Seq[Def.Setting[_]] = List()
   val bloopCompileGlobalSettings: Seq[Def.Setting[_]] = List()
-  val bloopCompileConfigSettings: Seq[Def.Setting[_]] = List(
-    BloopKeys.bloopCompile := Keys.compileIncremental.value
```
Comment (olafurpg) [inline integrations/sbt-bloop/src/main/scala-sbt-0.13/bloop/integrations/sbt/Offloader.scala]: Can you elaborate on why this change is safe?

Comment (tgodzik) [reply integrations/sbt-bloop/src/main/scala-sbt-0.13/bloop/integrations/sbt/Offloader.scala]: Not 100% sure, I tested it out on a `0.13` project and it worked. `Keys.compileIncremental` can  sometimes be not  there and it will break some projects when they include the Bloop plugin.

I will work on a test case for that though.

File: .gitignore [hunk 2]
```diff
@@ -33,6 +33,7 @@ website/build/bloop-gh-pages/
 node_modules/
 package-lock.json
 .metals/
+project/metals.sbt
```
Comment (jvican) [inline .gitignore]: Hmmm, Bloop loads the sbt-bloop plugin itself. You should not have this directory in your setup, otherwise bad things can happen (TM). The `-Dbloop.generate.sbt=false` option in `.jvmopts` is there so that Metals doesn't write this file and instead blindly assumes the plugin is already added in the project. Is that not being picked up somehow in your computer?

File: integrations/sbt-bloop/src/main/scala-sbt-0.13/bloop/integrations/sbt/Offloader.scala (hunk 1 above)
Comment (jvican) [reply integrations/sbt-bloop/src/main/scala-sbt-0.13/bloop/integrations/sbt/Offloader.scala]: I expect making this change is fine because we're not using `bloopCompile` directly in `SbtBloop` yet. I removed that to disable compile offloading. It's really surprising to me that `compileIncremental` is not scoped in sbt 0.13, most probably it's a bug.

**Comment by jvican** [comment]
Can you remove the .gitignore addition, push again and we merge? Feel free to merge when you do that, CI is failing right now but it's spurious.

File: .gitignore (hunk 2 above)
Comment (tgodzik) [reply .gitignore]: Och, that must have been an old file, sorry about that.

Comment (lionello) [reply .gitignore]: For the record: removing `project/metals.sbt` actually resolved the issue for me!
